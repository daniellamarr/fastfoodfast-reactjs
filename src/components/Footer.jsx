import React from 'react';

/**
 * Footer
 */
export default class Footer extends React.Component {
  /**
   * @returns {JSX} jsx
   */
  render() {
    return (
      <div>
        <footer>
          <div>
            <p>
              <a href="index.html">Fast-Food-Fast</a>
            </p>
            <p>
              Developed by
              <a
                href="https://github.com/daniellamarr"
                rel="noopener noreferrer" target="_blank"
              >
                Daniel Lamarr
              </a>
            </p>
          </div>
        </footer>
      </div>
    );
  }
}
