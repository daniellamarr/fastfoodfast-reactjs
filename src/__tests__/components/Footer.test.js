/* eslint-disable react/react-in-jsx-scope */
/* eslint-disable no-undef */
import { shallow } from 'enzyme';
import Footer from '../../components/Footer.jsx';

let wrapper;

beforeEach(() => {
  wrapper = shallow(<Footer />);
});

describe('Footer Component', () => {
  it('should match snapshot', () => {
    expect(wrapper).toMatchSnapshot();
  });
});
