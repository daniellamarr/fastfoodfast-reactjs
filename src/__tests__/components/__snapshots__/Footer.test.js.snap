// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer Component should match snapshot 1`] = `
<div>
  <footer>
    <div>
      <p>
        <a
          href="index.html"
        >
          Fast-Food-Fast
        </a>
      </p>
      <p>
        Developed by
        <a
          href="https://github.com/daniellamarr"
          rel="noopener noreferrer"
          target="_blank"
        >
          Daniel Lamarr
        </a>
      </p>
    </div>
  </footer>
</div>
`;
